src/pages/_document.tsx

```diff
@@ -0,0 +1,31 @@
+import Document from "next/document";
+import React from "react";
+import { ServerStyleSheet } from "styled-components";
+
+export default class MyDocument extends Document {
```

Comment:
MyDocument - > to brzmi jak skopiowane z getting starded -> raczej jakiś custom / app document czy coś